xla/service/gpu/cudnn_fused_conv_rewriter_test.cc

```diff
@@ -241,6 +252,19 @@ class CudnnFusedConvRewriterTest : public GpuCodegenTest {
   }
 };
 
+#if GOOGLE_CUDA
+  #if (CUDA_VERSION < 12000 || CUDNN_VERSION < 8900)
+    #define MAYBE_SKIP_TEST(CAUSE) \
+      if(absl::string_view(CAUSE) == "F8") \
+        GTEST_SKIP() << "FP8 convolutions require CUDA 12 and cuDNN 8.9.";
+  #else
+      #define MAYBE_SKIP_TEST(CAUSE)
+  #endif
+#else
+#define MAYBE_SKIP_TEST(CAUSE) \
```

Comment:
If you wrap the macro implementations in a `do { ... } while(0)`, then you can put a semicolon after the macro call, which looks more natural. E.g. `MAYBE_SKIP_TEST("F8");`
https://hownot2code.wordpress.com/2016/12/05/do-while-0-in-macros/

(I mean assuming that this works with gtest.)